taprpc/universerpc/universe.proto

```diff
@@ -126,6 +126,20 @@ service Universe {
     period, grouped by day.
     */
     rpc QueryEvents (QueryEventsRequest) returns (QueryEventsResponse);
+
+    /* tapcli `universe sync config`
```

Comment:
If this CLI sub command doesn't actually exist, this will break the API generator. So we should just remove the whole line. Or add the corresponding commands.

Reply:
I've removed for now, will add back in a future PR.